sky/skylet/constants.py

```diff
@@ -169,13 +176,14 @@
 _sky_version = str(version.parse(sky.__version__))
 RAY_STATUS = f'RAY_ADDRESS=127.0.0.1:{SKY_REMOTE_RAY_PORT} {SKY_RAY_CMD} status'
 RAY_INSTALLATION_COMMANDS = (
+    f'{SET_SKY_PIP} '
     'mkdir -p ~/sky_workdir && mkdir -p ~/.sky/sky_app;'
     # Print the PATH in provision.log to help debug PATH issues.
     'echo PATH=$PATH; '
     # Install setuptools<=69.5.1 to avoid the issue with the latest setuptools
     # causing the error:
     #   ImportError: cannot import name 'packaging' from 'pkg_resources'"
-    f'{SKY_UV_PIP_CMD} install "setuptools<70"; '
+    f'SKY_PIP install "setuptools<70"; '
```

Comment:
Nit: Should we just change the `SKY_UV_PIP_CMD` to  choose the right pip instead of creating the function? Not feeling strongly, just to avoid two different ways to use pip

Reply:
Same as above, this PR no longer touches pip installation commands